CHANGELOG.md

```diff
@@ -1,5 +1,81 @@
+* make writes to the backup file atomic ([#8566](https://github.com/confluentinc/ksql/pull/8566)) ([b113e9e](https://github.com/confluentinc/ksql/commit/b113e9ed4a81670424019dcff567aa01697ddca9))
+* only check configs on validation ([#8722](https://github.com/confluentinc/ksql/pull/8722)) ([6b65cd4](https://github.com/confluentinc/ksql/commit/6b65cd4da78e38832b6a9ec232595c3a8b8d1443))
+* pass DCN_NULLPOINTER_EXCEPTION spotbugs error ([#8775](https://github.com/confluentinc/ksql/pull/8775)) ([afd7fb1](https://github.com/confluentinc/ksql/commit/afd7fb1c72dc9ed52be3c991213277a401408bca))
+* prevent hanging stream pull queries on truncated topics ([#8740](https://github.com/confluentinc/ksql/pull/8740)) ([d66107c](https://github.com/confluentinc/ksql/commit/d66107ced9809b852e6c75935aa623499a793164))
+* re-order cache usage error ([#8909](https://github.com/confluentinc/ksql/pull/8909)) ([336c690](https://github.com/confluentinc/ksql/commit/336c690f81932fe263e224344a671fe71b85a0ab))
+* recreating old queries in the old runtime will still use the old runtimes ([#8704](https://github.com/confluentinc/ksql/pull/8704)) ([e248966](https://github.com/confluentinc/ksql/commit/e248966325099ba6043a0e3f9221be72e406813a))
+* reduce query creation time with shared runtimes ([#8888](https://github.com/confluentinc/ksql/pull/8888)) ([f3c5850](https://github.com/confluentinc/ksql/commit/f3c58508a379543b0e545a9cb03af1e9166760c3))
+* register schema within sandbox ([#8614](https://github.com/confluentinc/ksql/pull/8614)) ([ba572e0](https://github.com/confluentinc/ksql/commit/ba572e0e5bd8873e538d40a58153ba345eb1573d)), closes [#1394](https://github.com/confluentinc/ksql/issues/1394)
+* reinstate the old KsqlRestClient.create overload ([#8761](https://github.com/confluentinc/ksql/pull/8761)) ([ee4a1bc](https://github.com/confluentinc/ksql/commit/ee4a1bcf4ae4487e01bd4a6341feac402ffe6ddc))
+* reorder imports ([#8716](https://github.com/confluentinc/ksql/pull/8716)) ([30fd66b](https://github.com/confluentinc/ksql/commit/30fd66b3ecd26a9e37f2758b53a46d7fc514df4e))
+* resolve schema registry issue for pull query  ([#8876](https://github.com/confluentinc/ksql/pull/8876)) ([6a1c2ae](https://github.com/confluentinc/ksql/commit/6a1c2ae36d1257b050798b2a1f3cdbec4b68470d))
+* restore process fails due to DROP constraints ([#8803](https://github.com/confluentinc/ksql/pull/8803)) ([db070a2](https://github.com/confluentinc/ksql/commit/db070a211b51cfb9ef7138c9b05238ba41798f4b))
+* Set the sslFactory properly for the SR REST client. ([#8830](https://github.com/confluentinc/ksql/pull/8830)) ([e69a545](https://github.com/confluentinc/ksql/commit/e69a545a7765e9056da223075d4336d482b36614))
+* skip docker build for packaging ([#8717](https://github.com/confluentinc/ksql/pull/8717)) ([59f74e8](https://github.com/confluentinc/ksql/commit/59f74e808a63afb74f5d3c4185e3bf26f97537b6))

```

Comment:
Remove this -- not relevant for users. cc @nateab you can use "build" or "chore" instead of "fix" for these types of changes (non-user-relevant) in the future to avoid changelog entries being generated for them.